tests/unit/amo/components/TestPage.js

```diff
@@ -724,6 +726,209 @@ describe(__filename, () => {
+          utm_content: 'footer-link',
+          utm_campaign: null,
+        })}`,
+      );
+
+      expect(screen.getByRole('link', { name: 'Monitor' })).toHaveAttribute(
+        'href',
+        `https://monitor.firefox.com/${makeQueryStringWithUTM({
+          utm_content: 'footer-link',
+          utm_campaign: null,
+        })}`,
+      );
+
+      expect(screen.getByRole('link', { name: 'Browsers' })).toHaveAttribute(
+        'href',
+        `https://www.mozilla.org/firefox/browsers/${makeQueryStringWithUTM({
+          utm_content: 'footer-link',
+          utm_campaign: null,
+        })}`,
+      );
+
+      expect(screen.getByRole('link', { name: 'Pocket' })).toHaveAttribute(
+        'href',
+        `https://getpocket.com${makeQueryStringWithUTM({
+          utm_content: 'footer-link',
+          utm_campaign: null,
+        })}`,
+      );
+
+      expect(screen.getByRole('link', { name: 'Desktop' })).toHaveAttribute(
+        'href',
+        `https://www.mozilla.org/firefox/new/${makeQueryStringWithUTM({
+          utm_content: 'footer-link',
+          utm_campaign: null,
+        })}`,
+      );
+
+      expect(screen.getByRole('link', { name: 'Mobile' })).toHaveAttribute(
+        'href',
+        `https://www.mozilla.org/firefox/mobile/${makeQueryStringWithUTM({
+          utm_content: 'footer-link',
+          utm_campaign: null,
+        })}`,
+      );
+
+      expect(screen.getByRole('link', { name: 'Enterprise' })).toHaveAttribute(
+        'href',
+        `https://www.mozilla.org/firefox/enterprise/${makeQueryStringWithUTM({
+          utm_content: 'footer-link',
+          utm_campaign: null,
+        })}`,
+      );
+
+      expect(
+        screen.getByRole('link', { name: 'Developer Policies' }),
+      ).toHaveAttribute(
+        'href',
+        `${extensionWorkshopUrl}/documentation/publish/add-on-policies/${makeQueryStringWithUTM(
+          {
+            utm_medium: 'photon-footer',
+            utm_campaign: null,
+          },
+        )}`,
+      );
+
+      expect(
+        screen.getByRole('link', { name: 'Community Blog' }),
+      ).toHaveAttribute(
+        'href',
+        `https://blog.mozilla.com/addons${makeQueryStringWithUTM({
+          utm_campaign: null,
+          utm_content: 'footer-link',
+          utm_medium: 'referral',
+        })}`,
+      );
+    });
+  });
+
+  describe('Tests for LanguagePicker', () => {
+    it('renders a LanguagePicker', () => {
+      render();
+
+      expect(
+        screen.getByRole('combobox', { name: 'Change language' }),
+      ).toBeInTheDocument();
+    });
+
+    it('selects the current locale', () => {
+      store = dispatchClientMetadata({ lang: 'fr' }).store;
+      render({ location: '/fr/firefox/' });
+
+      expect(screen.getByRole('option', { name: 'Français' }).selected).toEqual(
+        true,
+      );
+      expect(screen.getByRole('option', { name: 'Français' }).value).toEqual(
```

Comment:
I might be wrong about this, but if the `fr` option is selected, that I would think that the value of the combobox would be `fr`, so maybe, instead of checking the value of this option, we should check the value of the comboxbox?